ruby/vampires.rb

```diff
@@ -0,0 +1,129 @@
+# DBC 4.4
+# Interview with the Vampire
+
+# Ask the user how many employees will be processed.
+# After the result is printed for the first employee, start over and repeat
+# until all the employees have been processed. 
+
+puts "How many employees are you processing today?"
+employees = gets.chomp
+
+number = 0
+while number < employees.to_i 
+  number += 1
+
+  # Ask a new employee the following questions: 
+  # What is your name? 
+  # How old are you? 
+  # What year were you born? 
+  # Our company cafeteria serves garlic bread. Would you like us to order some for you? 
+  # Would you like to enroll in the company's health insurance?
+  
+  puts "What is your name?"
+  name = gets.chomp
+  
+    # Do they have a vampire name??
+
+    if name == "Drake Cula"
```

Comment:
You can combine the `if` and `elsif` using the `||` operator. 

`if name == "Tu Fang" || name == "Drake Cula"`